prebid-mobile/pbm-api/ios/code-integration-ios.md

```diff
@@ -221,6 +221,8 @@ The `Prebid` class is a singleton that enables the user to apply global settings
 
 `prebidServerHost`: String containing configuration your Prebid Server host with which Prebid SDK will communicate. Choose from the system-defined Prebid Server hosts or define your own custom Prebid Server host.
 
+`auctionSettingsId`: String to separate out account from "auction settings". There is used to set `ext.prebid.storedrequest.id`, otherwise prebidServerAccountId is taken by default.
```

Comment:
```suggestion
`auctionSettingsId`: Allows you to separate account from "auction settings". This is used to set `ext.prebid.storedrequest.id`, otherwise prebidServerAccountId is taken by default. This allows each app to have different global parameters like timeout, price granularity, etc. Please work with your Prebid Server provider to determine what to enter here. 
```